tests/test_api_v1.py

```diff
@@ -371,7 +375,7 @@ def test_set_deps(app, client, db, worker_auth_env, sample_deps):
     db.session.add(request)
     db.session.commit()
 
-    payload = {'dependencies': sample_deps}
+    payload = {'dependencies': sample_deps, 'environment_variables': env_vars}
```

Comment:
Could you please verify the environment variables are actually recorded?

Reply:
Good catch! I also added a  verification to ensure the environment variables in the request json actually maps to the expected value.